lib/ferrum/utils/stream.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module Ferrum
+  module Utils
+    module Stream
+      module_function
+
+      def for(source)
```

Comment:
That's very very bad to reassign things like that in a module like this

Reply:
okay, good to know :+1: "if you don't try, you won't know", so let's refactor it.